app/src/App.js

```diff
@@ -26,9 +27,29 @@ const moonPhaseUrl = `https://www.icalendar37.net/lunar/api/?lang=en&month=${
 //   console.log(data);
 //   return data;
 // };
-
+const calculateMoonPhase = (num) => {
+  const decimal = Math.round(num);
+  if (decimal === 0) {
+    return "New Moon";
+  } else if (decimal > 0 && decimal < 25) {
```

Comment:
```suggestion
  } else if (decimal > 0 && decimal <= 25) {
```
I don't believe the case of being equal to 25 is ever caught in this block - may want to adjust this to say less than or equal to